client/components/cards/attachments.js

```diff
@@ -38,4 +65,47 @@ Template.cardAttachmentsPopup.events({
     tpl.find('.js-attach-file').click();
     evt.preventDefault();
   },
+  'click .js-upload-clipboard-image': Popup.open('previewClipboardImage'),
+});
+
+Template.previewClipboardImagePopup.onRendered(function() {
+  // we can paste image from clipboard
+  $('html').pasteImageReader(function(results) {
+    if(results.dataURL.startsWith('data:image/')) {
+      Popup.pasted = results;
+      $('img.preview-clipboard-image').attr('src', results.dataURL);
+    }
+  });
+
+  // we can also drag & drop image file to it
+  $('html').dropImageReader(function(results) {
+    if(results.dataURL.startsWith('data:image/')) {
+      Popup.pasted = results;
```

Comment:
You should use a local variable and not pollute the Popup object (which is too fragile and will cause bugs). Just create something `let droppedImage;` on top of this file, and use it here.